auth_admin_passkey/model/__init__.py

```diff
@@ -1,24 +1,8 @@
-# -*- encoding: utf-8 -*-
-##############################################################################
-#
-#    Admin Passkey module for OpenERP
-#    Copyright (C) 2013-2014 GRAP (http://www.grap.coop)
-#    @author Sylvain LE GAL (https://twitter.com/legalsylvain)
```

Comment:
I just checked the other migrated modules, many of them had aditional lines, but they were all deleted. Only the copyright line has been kept... They think the autor field in readme and manifest file is eneugh, since its shown in odoo in the modules description.